src/ngMock/angular-mocks.js

```diff
@@ -1341,13 +1341,13 @@ function createHttpBackendMock($rootScope, $timeout, $delegate, $browser) {
       // service the ngMockE2E version can pass through to the real backend
       originalHttpBackend = $delegate.$$originalHttpBackend || $delegate;
 
-  function createResponse(status, data, headers, statusText) {
+  function createResponse(status, data, headers, statusText, xhrStatus) {
     if (angular.isFunction(status)) return status;
 
     return function() {
       return angular.isNumber(status)
-          ? [status, data, headers, statusText]
-          : [200, status, data, headers];
+          ? [status, data, headers, statusText, xhrStatus || (status === 200 ? 'success' : 'error')]
```

Comment:
LOL - I contradict myself in these comments: https://github.com/angular/angular.js/pull/15847#discussion_r107878049
https://github.com/angular/angular.js/pull/15847#discussion_r107919244
It doesn't make sense to make it configurable imo, so let's remove the `xhrStatus` argument and make it `'complete'` everywhere.